src/test/resources/data/AutomatedSessionRemindersTest.json

```diff
@@ -22,7 +22,7 @@
       "googleId": "AutSessRem.instructor",
       "courseId": "AutSessRem.course",
       "name": "Test Ins for Aut Sessions Reminder",
-      "email": "${support.email}",
+      "email": "${test.instructor}@gmail.com",
```

Comment:
I'm not saying that the same email account should be used for `${test.instructor}`. You can simply check the `${test.instructor}` email box, whatever account used for it, to see if the emails arrive. I'm not sure if checking more than one email boxes (if it's not already done that way) is more troublesome than having the test slowed down because of the extra `.replace(...)`.